automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -41,8 +44,10 @@ trait GPAllocBeforeAndAfterAll extends BeforeAndAfterAll with BillingFixtures wi
 
   override def beforeAll(): Unit = {
     super.beforeAll()
-    val billingProject = claimProject()
-    sys.props.put(gpallocProjectKey, billingProject.value)
+    Either.catchNonFatal(claimProject()) match {
+      case Left(e) => sys.props.put(gpallocProjectKey, "Failed To Claim Project: "+e.getMessage)
```

Comment:
Oh I see that's our message - thought it came from an external library or something.